src/ecdsa/keys.py

```diff
@@ -1281,6 +1293,19 @@ def to_der(
                 der.encode_octet_string(ec_private_key),
             )
 
+    def to_ssh(self):
```

Comment:
what about `from_ssh`?

Reply:
I am focusing only on exporting for now, is that an issue?

Reply:
every other key operation has support for both import and export. And for test coverage it would be good to not only test against static vectors, but also to verify that a random key can be imported and exported.